components/appmgt/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/dao/AppMDAO.java

```diff
@@ -5988,19 +5962,14 @@ private long getApplicationAccessTokenValidityPeriod() {
 
 	}
 
-	public APIKeyValidationInfoDTO getApplicationData(String appContext, String appVersion,
-													  String consumer, AuthenticatedIDP[] authenticatedIDPs) throws
-			AppManagementException {
-
-		Connection connection = null;
-
-		try {
-
-			connection = APIMgtDBUtil.getConnection();
-
-			boolean hasValidSubscription = false;
-
-			Subscription subscription = getIndividualSubscription(consumer, appContext, appVersion, connection);
+    public APIKeyValidationInfoDTO getApplicationData(String appContext, String appVersion, String consumer,
```

Comment:
Better to have method comments in public methods which are being consumed by other components.